ludwig/features/numerical_feature.py

```diff
@@ -49,26 +50,36 @@ class NumericalFeatureMixin(object):
 
     @staticmethod
     def get_feature_meta(column, preprocessing_parameters):
-        if preprocessing_parameters['normalization'] is not None:
-            if preprocessing_parameters['normalization'] == 'zscore':
-                return {
-                    'mean': column.astype(np.float32).mean(),
-                    'std': column.astype(np.float32).std()
-                }
-            elif preprocessing_parameters['normalization'] == 'minmax':
-                return {
-                    'min': column.astype(np.float32).min(),
-                    'max': column.astype(np.float32).max()
-                }
-            else:
-                logger.info(
-                    'Currently zscore and minmax are the only '
-                    'normalization strategies available. No {}'.format(
-                        preprocessing_parameters['normalization'])
-                )
-                return {}
-        else:
-            return {}
+        numeric_transformer = get_from_registry(
+            preprocessing_parameters.get('normalization', None),
+            numeric_transformation_registry
+        )(**preprocessing_parameters)
```

Comment:
As `fit_transform_params` is a static method, we don't need `(**preprocessing_parameters)`.

Reply:
done